### PR TITLE
Closes #1217: Update social metadata block

### LINKED ISSUE #1217: Update social science metadata block

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: dataverse-webapp/src/main/resources/socialscience_en.properties [hunk 1]
```diff
@@ -1,85 +1,373 @@
...
+datasetfieldtype.interviewsBackcheckPercentage.title=Percentage of Interviews
+datasetfieldtype.interviewsBackcheckPercentage.withParent.title=Percentage of Back-Checked Interviews
+datasetfieldtype.interviewsBackcheckProcedureDescription.title=Back-Check Procedure Description
+datasetfieldtype.interviewsBackcheckProcedureDescription.withParent.title=Descritpion of Back-Check Procedure
 datasetfieldtype.cleaningOperations.title=Cleaning Operations
 datasetfieldtype.datasetLevelErrorNotes.title=Study Level Error Notes
 datasetfieldtype.responseRate.title=Response Rate
 datasetfieldtype.samplingErrorEstimates.title=Estimates of Sampling Error
-datasetfieldtype.otherDataAppraisal.title=Other Forms of Data Appraisal
+datasetfieldtype.otherDataAppraisal.title=otherDataAppraisal
```
Comment (tomaszblazejczyk) [inline dataverse-webapp/src/main/resources/socialscience_en.properties]: Is this title correct?

Comment (madryk) [reply dataverse-webapp/src/main/resources/socialscience_en.properties]: No - It shouldn't be the same as original

File: dataverse-webapp/src/main/resources/socialscience_en.properties [hunk 2]
```diff
@@ -31,55 +56,318 @@ datasetfieldtype.socialScienceNotesSubject.title=Subject
...
+datasetfieldtype.dataSourceType.description=Data source type for this dataset. 
+datasetfieldtype.samplingDescription.description=Detailed description of the sample and sample design used to select the survey respondents to represent the population. May include reference to the target sample size and the sampling fraction.
+datasetfieldtype.samplingProcedure.description=Type of sampling procedure used to select the survey respondents to represent the population.
+datasetfieldtype.targetSampleSize.description=Specific information regarding the target sample size and achieved sample size.
+datasetfieldtype.targetSampleActualSize.description=Target sample size.
+datasetfieldtype.targetSampleSizeAchieved.description=Achieved sample size. 
+datasetfieldtype.deviationsFromSampleDesign.description=Indicate deviations in statistics between the sample and the population (regarding age, sex, marital status etc.)
+datasetfieldtype.substitutionOfRespondents.description=Infromation regarding the substitution of respondents.
```
Comment (tomaszblazejczyk) [inline dataverse-webapp/src/main/resources/socialscience_en.properties]: typo

File: dataverse-webapp/src/main/resources/socialscience_en.properties [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
 datasetfieldtype.dataCollectionSituation.title=Characteristics of Data Collection Situation
+datasetfieldtype.weighting.title=Weight
 datasetfieldtype.actionsToMinimizeLoss.title=Actions to Minimize Losses
 datasetfieldtype.controlOperations.title=Control Operations
-datasetfieldtype.weighting.title=Weighting
+datasetfieldtype.interviewsBackcheck.title=Interviews' Back-Check
+datasetfieldtype.interviewsBackcheckPercentage.title=Percentage of Interviews
+datasetfieldtype.interviewsBackcheckPercentage.withParent.title=Percentage of Back-Checked Interviews
+datasetfieldtype.interviewsBackcheckProcedureDescription.title=Back-Check Procedure Description
+datasetfieldtype.interviewsBackcheckProcedureDescription.withParent.title=Descritpion of Back-Check Procedure
```
Comment (tomaszblazejczyk) [inline dataverse-webapp/src/main/resources/socialscience_en.properties]: typo

File: dataverse-webapp/src/main/resources/socialscience_en.properties [hunk 4]
```diff
@@ -31,55 +56,318 @@ datasetfieldtype.socialScienceNotesSubject.title=Subject
...
-datasetfieldtype.weighting.description=The use of sampling procedures might make it necessary to apply weights to produce accurate statistical results. Describes the criteria for using weights in analysis of a collection. If a weighting formula or coefficient was developed, the formula is provided, its elements are defined, and it is indicated how the formula was applied to the data.
-datasetfieldtype.cleaningOperations.description=Methods used to clean the data collection, such as consistency checking, wildcode checking, or other.
-datasetfieldtype.datasetLevelErrorNotes.description=Note element used for any information annotating or clarifying the methodology and processing of the study.
-datasetfieldtype.responseRate.description=Percentage of sample members who provided information.
+datasetfieldtype.controlOperations.description=Methods to facilitate data control performed by the primary investigator or by the data archive.
+datasetfieldtype.interviewsBackcheck.description=Description of back-check control of completed interviews with respect to the fact and correctnes of their fulfillment, eg. by coordinator.
```
Comment (tomaszblazejczyk) [inline dataverse-webapp/src/main/resources/socialscience_en.properties]: typo

**Review by tomaszblazejczyk — APPROVED** [review]

**Pull request merged**